http://bugs.gentoo.org/308649

asgtklookedit.c:658:3: warning: implicit declaration of function ‘make_session_data_file’

--- libASGTK/asgtklookedit.c
+++ libASGTK/asgtklookedit.c
@@ -28,6 +28,7 @@
 #include "../libAfterStep/parser.h"
 #include "../libAfterStep/freestor.h"
 #include "../libAfterStep/mystyle.h"
+#include "../libAfterStep/session.h"
 #include "../libAfterConf/afterconf.h"
 
 #include <unistd.h>		   
